ecommerce-ios/Sources/Home/HomeScreen.swift

```diff
@@ -13,13 +13,20 @@ struct HomeScreen: View {
 
     var body: some View {
         NavigationView {
-            Text("Home")
-                .navigationBarTitle(tab.title)
-                .navigationBarLargeTitleItems(trailing: Button(action: {
-                    print("did tap profile button")
-                }) {
-                    Image("avatar-icon")
-                })
+            ScrollView {
+                VStack {
+                    SuggestedView()
+                        .cornerRadius(12.0)
+                        .offset(x: 0.0, y: 20.0)
+                        .padding([.horizontal], 10.0)
+                }
+            }
+            .navigationBarTitle(tab.title)
+            .navigationBarLargeTitleItems(trailing: Button(action: {
+                print("did tap profile button")
+            }) {
+                Image("avatar-icon")
+            })
```

Comment:
Because you use initializer with 2 closure, so I suggest to use full closure syntax instead of trailing syntax.

```suggestion
            .navigationBarLargeTitleItems(
                trailing: Button(
                    action: { print("did tap profile button") },
                    content: { Image("avatar-icon") }
                )
             )
```